test/controllers/issues_controller_test.rb

```diff
@@ -18,4 +18,57 @@ class IssuesControllerTest < ActionDispatch::IntegrationTest
     assert_equal test_line_id, issue.line_onestop_id
     assert_response :success
   end
+
+  test "does not allow for visitor to create issue" do
```

Comment:
Clarify the wording here to make it clear who cannot create an issue. I'd suggest "Unauthenticated user cannot create issue", as it conveys the reason why an issue should not be created here.